rllib/evaluation/worker_set.py

```diff
@@ -108,53 +105,6 @@ def __init__(
             self._local_worker = None
             if num_workers == 0:
                 local_worker = True
-            if (
```

Comment:
Do we not need this logic for backward compatibility and throwing informative errors?

Reply:
no this already exists in the json reader so it should be fine/further down in this file actually.